database/src/pool.rs

```diff
@@ -980,9 +980,28 @@ mod tests {
 
             db.mark_benchmark_request_as_completed(tag).await.unwrap();
 
-            let completed = db.load_benchmark_request_index().await.unwrap();
+            /* From the status page view we can see that the duration has been
+             * updated. Albeit that it will be a very short duration. */
+            let status_page_view = db.get_status_page_data().await.unwrap();
```

Comment:
You can add a sleep to the test to wait e.g. at least 1 second.

Reply:
43c0c4b7e8d7e10e6cea24cb92b37d2d86b2edd0 :+1: